### PR TITLE
feat: storybook addons

### PR DESCRIPTION
**Pull request by gabewomble**
This PR introduces a couple new Storybook addons and fixes the theming in storybook ([solution here](https://github.com/chakra-ui/chakra-ui/issues/2527#issuecomment-728161743))

Addons (screenshots are from our storybook):

[a11y](https://storybook.js.org/addons/@storybook/addon-a11y)

Screen Shot 2021-06-04 at 3 22 28 PM

[storysource](https://storybook.js.org/addons/@storybook/addon-storysource)

Screen Shot 2021-06-04 at 3 23 28 PM

[performance](https://storybook.js.org/addons/storybook-addon-performance)

Screen Shot 2021-06-04 at 3 25 19 PM

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+  ],
+  webpackFinal: async (config) => {
+    return {
+      ...config,
+      resolve: {
+        ...config.resolve,
+        alias: {
+          ...config.resolve.alias,
+          "@emotion/core": toPath("node_modules/@emotion/react"),
+          "emotion-theming": toPath("node_modules/@emotion/react")
```
Comment (MrArnoldPalmer) [inline .storybook/main.js:25]: This is really funky, but I guess makes sense.

File: .storybook/preview.js [hunk 2]
```diff
@@ -1,3 +1,16 @@
+import { withPerformance } from "storybook-addon-performance";
+import { Theme } from "../src/Theme";
+import { CSSReset } from "@chakra-ui/react";
+
+export const decorators = [
+  withPerformance,
+  (Story) => (
+    <Theme>
+      <CSSReset />
```
Comment (MrArnoldPalmer) [inline .storybook/preview.js:9]: We don't currently use the reset in the app right? We probably should.

**Review by MrArnoldPalmer — APPROVED** [review]

File: .storybook/preview.js (hunk 2 above)
Comment (gabewomble) [reply .storybook/preview.js:9]: Good callout - I'll bring that in too

File: .storybook/main.js (hunk 1 above)
Comment (gabewomble) [reply .storybook/main.js:25]: Yeah, it's weird. I linked it in the description above, but found this solution from the [issue here](https://github.com/chakra-ui/chakra-ui/issues/2527#issuecomment-728161743).

**Pull request merged**